plugins/org.mozilla.Firefox.Plugin.Flash.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: org.mozilla.Firefox.Plugin.Flash
```

Comment:
Wonder if we should use the same name used by the unofficial firefox flatpak at https://github.com/ozoder/firefox-devedition-flatpak/tree/master/com.adobe.FlashPlayer.NPAPI.

Also add missing appdata for it (as done in the repo above).

Reply:
Worth noting that the common prefix has to be shared by all plugins, so if e.g. there were another plugin that were to be provided via this path (e.g. codecs or something), they would also need to have the Flash-specific prefix.

Reply:
That might be over-generalisation considering now I think the only NPAPI plugin that Firefox would ever load is Flash, and only for maybe 1-2 years more. So actually hardcoding the Firefox flatpak to a specific Flash extension point might be fine.